docs/debugging.md

```diff
@@ -0,0 +1,6 @@
+# Debugging Helpers
+
+- To log all database requests for a specific key: `RUST_LOG="fedimint_api[{key=ContractKey.*}]=debug"`. 
```

Comment:
I do not immediately understand this syntax. Please give an example and maybe link to a doc of the crate that describes how it works.